rllib/core/learner/tests/test_learner_group.py

```diff
@@ -363,6 +366,84 @@ def test_save_load_state(self):
                 weights_after_1_update_with_break, weights_after_1_update_without_break
             )
 
+    def test_load_module_state(self):
```

Comment:
Just want to point out that the resources / time required to run this test may increase after adding this unittest, If that happens we may have to break the test down to smaller isolated unittest.